src/api/timetable/getGroups.ts

```diff
@@ -0,0 +1,11 @@
+import { useAPI } from '@/api/api';
+import { useState } from 'react';
+
+export function useGroups() {
+  const api = useAPI();
+  const [groups, setGroups] = useState<string[]>([]);
+  /*useEffect(() => {
+    api.get<string[]>('/timetable/groups/').on(StatusCodes.OK, setGroups);
```

Comment:
Sauf si on modifie la route pour faire moins long/a-t-on besoin du current ?
```suggestion
    api.get<string[]>('/timetable/current/groups/').on(StatusCodes.OK, setGroups);
```
Edit : btw pourquoi ça a été commenté ?
> Pour l'API: _Alors oui il faudra générer les groupes lors de la création des utilisateurs. D'ailleurs, on pourrait imaginer des triggers pour chaque module, des fonctions qui seraient appelés lorsque certains évènements spécifiques surviennent (ici, la création d'un utilisateur; mais aussi la création d'une ue, etc etc)_

Reply:
C'était potentiellement pour une raison du style "la route n'existe pas" je pense